### DISCUSSION TITLE
Media uploads not possible anymore on my instance without having changed anything

### DISCUSSION THREAD

**Discussion by braydofficial** [mastodon/mastodon#17661] [Q&A]
### Steps to reproduce the problem

Try to upload media for a post / change your profile picture etc. (Worked yesterday, doesn't work since a few hours anymore). Restarting the server did not help. Uninstalling and reinstalling imagemagick didn't help.

### Expected behaviour

Media should be uploaded

### Actual behaviour

You get an 500 Error processing thumbnail for uploaded media

### Specifications

v3.4.6

Ubuntu Server Kernel 5.4.0
CPU: Intel Xeon E5-2630 0 (10) @ 1.9
Memory running at an average of: 3675MiB / 49152MiB

**Comment by Gargron** [comment]
Please provide error logs or we cannot help you

https://docs.joinmastodon.org/admin/troubleshooting/#i-see-an-error-page-that-says-something-went-wrong-how-do-i-find-out-whats-wrong

**Reply by ghost** [reply]
I'm experiencing the same on 3.5.3, not seeing errors on logs, but the upload is stuck.

![Screenshot 2022-11-06 at 10 25 54](https://user-images.githubusercontent.com/663841/200173360-0595acd4-01b6-408a-b710-578dd587ebc3.png)

**Reply by ghost** [reply]
I think it's related to the sidekiq queue.....

**Comment by gabrix73** [comment]
Same experience on my end.
